storage/spanstore/metrics/decorator_test.go

```diff
@@ -33,13 +33,13 @@ func TestSuccessfulUnderlyingCalls(t *testing.T) {
 
 	mockReader := mocks.Reader{}
 	mrs := NewReadMetricsDecorator(&mockReader, mf)
-	mockReader.On("GetServices").Return([]string{}, nil)
+	mockReader.On("GetServices", context.Background()).Return([]string{}, nil)
```

Comment:
how did this ever work before?

Reply:
My hypothesis is that the field is ret reflectively and the prior absence of this field in both the `On` invocation and the generated code failed silently. (I don't understand how the mock could satisfy the reader interface though.)